plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/RaptorQueryRunner.java

```diff
@@ -55,6 +57,20 @@ public static DistributedQueryRunner createRaptorQueryRunner(
             boolean bucketed,
             Map<String, String> extraRaptorProperties)
             throws Exception
+    {
+        List<TpchTable<?>> tablesToLoad = ImmutableList.of();
+        if (loadTpch) {
+            tablesToLoad = TpchTable.getTables();
+        }
```

Comment:
we should remove the `createRaptorQueryRunner` overload taking `loadTpch`, and simply pass empty list of tables when no tables needed

Reply:
if many places to update -- just add a TODO note in the code

Reply:
done